examples/jax/debugging/simple_spread/recurrent/decentralised/run_ippo.py

```diff
@@ -74,17 +82,21 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
         time_delta=log_every,
     )
 
-    # Optimisers.
+    # Optimizer.
     policy_optimiser = optax.chain(
-        optax.clip_by_global_norm(40.0), optax.scale_by_adam(), optax.scale(-3e-4)
+        optax.clip_by_global_norm(40.0), optax.scale_by_adam(), optax.scale(-1e-4)
     )
+
     critic_optimiser = optax.chain(
-        optax.clip_by_global_norm(40.0), optax.scale_by_adam(), optax.scale(-1e-3)
+        optax.clip_by_global_norm(40.0), optax.scale_by_adam(), optax.scale(-1e-4)
     )
 
-    # Create the system.
+    # Initialise the feedforward IPPO system.
```

Comment:
We can have a chat about this at some point. Yes, that is how I originally did it. But then I thought it would be nice to show the ways one could easily update the default IPPO. If we have more component groups we could mix and match by just adding the groups we currently care about, e.g. default system + recurrent components + checkpointer components. But it is super easy to create a recurrent IPPO default system. Let me know if that would be better?

Reply:
After discussing this, I agree that the current method is better. Thanks